alf/networks/preprocessors.py

```diff
@@ -75,13 +75,20 @@ def __init__(self,
             self._embedding_net = nn.Embedding(N, embedding_dim)
         else:
             # Only use an MLP for embedding a continuous input
+            # Manually specify all arguments to avoid being overwritten by gin
+            # configuration accidentally
             self._embedding_net = alf.networks.EncodingNetwork(
                 input_tensor_spec=input_tensor_spec,
+                input_preprocessors=None,
+                preprocessing_combiner=None,
                 conv_layer_params=conv_layer_params,
                 fc_layer_params=fc_layer_params,
                 activation=activation,
+                kernel_initializer=None,
                 last_layer_size=embedding_dim,
-                last_activation=last_activation)
+                last_activation=last_activation,
+                last_kernel_initializer=None,
+                name="preprocessor_embedding_net")
```

Comment:
Do we want to allow the possibility of changing kernel_initializer or last_kernel_initializer?

Reply:
Yeah I removed kernel initializers